webpack/global_index.js

```diff
@@ -23,3 +24,4 @@ addGlobalFill('host-details-page-tabs', 'Content', <ContentTab key="content" />,
 addGlobalFill('host-details-page-tabs', 'Traces', <TracesTab key="traces" />, 100);
 
 addGlobalFill('details-cards', 'Content View Details', <ContentViewDetailsCard key="content-view-details" />);
+addGlobalFill('details-cards', 'Content View Details', <ErrataOverviewCard key="errata-overview" />);
```

Comment:
If the name isn't unique apparently it erases the previous card.

```suggestion
addGlobalFill('details-cards', 'Installable errata', <ErrataOverviewCard key="errata-overview" />);
```